plugin/audit_log/audit_log.c

```diff
@@ -246,6 +247,36 @@ void csv_escape(const char *in, size_t *inlen, char *out, size_t *outlen)
   escape_buf(in, inlen, out, outlen, rules);
 }
 
+static const escape_buf_func_t format_escape_func[]=
+  { xml_escape, xml_escape, json_escape, csv_escape };
+
+/*
+  Calculate the size of the otput bufer needed to escape the string.
+
+  @param[in]  in           Input string
+  @param[in]  len          Length of the input string
+
+  @return
+    size of the otput bufer including trailing zero
+*/
+static
+size_t escape_string_dry_run(const char *in, size_t len)
```

Comment:
A name like "calculate_escape_string_buf_len" would be more descriptive what the function does (and matches the header comment even) whereas "escape_string_dry_run" is more how it's implemented